### PR TITLE
Feature/contacts/95 account page

### PR DESCRIPTION
**Pull request by AnnaKurochkina**
Completed  a single AccountPage from Contact section. 
https://trello.com/c/2PNpOb1b/95-create-individual-account-pages-when-a-contact-item-is-clicked

Created BankInfoSquare component for single bank details.
https://trello.com/c/jEpH4kBd/96-create-a-component-for-the-user-on-contact-account-details-page

Updated RecentTransactions component.

### CODE REVIEW

File: src/components/LoggedIn/LoggedIn.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ const LoggedIn = ({firstName="Samantha", lastName="Brooks"}) => {
     
     return (
         <div className="userProfile">
-            <p className="userProfile__name">{firstName} {lastName}</p>
+            <p className="userProfile__name">{firstName} {lastName} Samantha Brooks</p>
```
Comment (lanniiee) [inline src/components/LoggedIn/LoggedIn.jsx:12]: remove hardcoding

File: .eslintrc.json [hunk 2]
```diff
@@ -15,7 +15,8 @@
   "rules": {
     "react/jsx-uses-react": "error",
     "react/react-in-jsx-scope": "off",
-    "react/prop-types": 0
+    "react/prop-types": 0,
+    "no-unused-vars": "off"
```
Comment (lanniiee) [inline .eslintrc.json:19]: is there any reason why this was added?

File: src/components/BankInfoSqure/BankInfoSquare.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import "./BankInfoSquare.scss";
+
+
+
+const BankInfoSqaure = ({firstName,lastName,img,sortCode,accNum,iban}) => {
+  firstName="erik";lastName="drew";img="test";sortCode="2728729";accNum="929292929929292";iban="7478489383838838"
```
Comment (kirsty-nology) [inline src/components/BankInfoSqure/BankInfoSquare.jsx:6]: remove hard coding

File: src/components/BankInfoSqure/BankInfoSquare.jsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+  return(
+    <>
```
Comment (kirsty-nology) [inline src/components/BankInfoSqure/BankInfoSquare.jsx:9]: unnecessary empty wrapper

File: src/components/BankInfoSqure/BankInfoSquare.jsx (hunk 3 above)
Comment (ChrisWakley) [inline src/components/BankInfoSqure/BankInfoSquare.jsx:6]: Please remove hard coded props as they are unnecessary in the live site.

File: src/components/BankInfoSqure/BankInfoSquare.jsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+      <div className = "container">
```
Comment (kirsty-nology) [inline src/components/BankInfoSqure/BankInfoSquare.jsx:10]: `className="bankContainer"`

File: src/components/BankInfoSqure/BankInfoSquare.jsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        <div className="row" id="row-one">
```
Comment (kirsty-nology) [inline src/components/BankInfoSqure/BankInfoSquare.jsx:11]: `className="bankContainer__row"`

File: src/components/BankInfoSqure/BankInfoSquare.jsx [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+          <div className="inner-left-row"><p>{firstName} {lastName}</p></div>
```
Comment (kirsty-nology) [inline src/components/BankInfoSqure/BankInfoSquare.jsx:12]: `className="bankContainer__leftRow"`

File: src/components/BankInfoSqure/BankInfoSquare.jsx [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+          <p className="heading">{img}</p>
```
Comment (kirsty-nology) [inline src/components/BankInfoSqure/BankInfoSquare.jsx:13]: `className="bankContainer__heading"`

File: src/components/BankInfoSqure/BankInfoSquare.jsx [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+        </div>
+        <div className="row">
```
Comment (kirsty-nology) [inline src/components/BankInfoSqure/BankInfoSquare.jsx:15]: as above

File: src/components/BankInfoSqure/BankInfoSquare.jsx [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+          <p className="heading">Sort Code</p>
```
Comment (kirsty-nology) [inline src/components/BankInfoSqure/BankInfoSquare.jsx:16]: as above

File: src/components/BankInfoSqure/BankInfoSquare.jsx [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+          <p>{sortCode}</p>
+        </div>
+        <div className="row">
```
Comment (kirsty-nology) [inline src/components/BankInfoSqure/BankInfoSquare.jsx:19]: as above

File: src/components/BankInfoSqure/BankInfoSquare.jsx [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+          <p className="heading">Account No</p>
```
Comment (kirsty-nology) [inline src/components/BankInfoSqure/BankInfoSquare.jsx:20]: as above

File: src/components/BankInfoSqure/BankInfoSquare.jsx [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+          <p>{accNum}</p>
+        </div>
+        <div className="row">
```
Comment (kirsty-nology) [inline src/components/BankInfoSqure/BankInfoSquare.jsx:23]: as above

File: src/components/BankInfoSqure/BankInfoSquare.jsx [hunk 14]
```diff
@@ -0,0 +1,32 @@
...
+          <p className="heading">IBAN</p>
```
Comment (kirsty-nology) [inline src/components/BankInfoSqure/BankInfoSquare.jsx:24]: as above

File: src/components/BankInfoSqure/BankInfoSquare.scss [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+            font-size: 1.5rem;
+            font-weight: 500;
+            line-height: 1.8rem;
+            color: $color-heading;
+            text-align: start;
+        }  
+    }
+}
+    #row-one{
+
```
Comment (kirsty-nology) [inline src/components/BankInfoSqure/BankInfoSquare.scss:52]: remove extra line

File: src/pages/AccountPage/AccountPage.scss [hunk 16]
```diff
@@ -0,0 +1,93 @@
...
+            display: grid;
+            grid-template-columns: 30% 70%;
+            grid-template-rows: 80% 20%;
+            grid-template-areas: 
+                "contact transactions"
+                "transfer transactions";
+            grid-gap: 1rem;
+            justify-items: center;
+
+        .accountWrapper__transferRow {
```
Comment (ChrisWakley) [inline src/pages/AccountPage/AccountPage.scss:36]: &__transferRow

File: src/pages/AccountPage/AccountPage.scss [hunk 17]
```diff
@@ -0,0 +1,93 @@
...
+            height: 100%;
+            padding: 1rem 2rem;
+            background-color: $color-currency-background;
+            grid-area: transfer;
+            p { 
+            color: $color-heading;
+            }
+        }
+
+        .accountWrapper__singleContact {
```
Comment (ChrisWakley) [inline src/pages/AccountPage/AccountPage.scss:49]: &__singleContact

File: src/pages/AccountPage/AccountPage.scss [hunk 18]
```diff
@@ -0,0 +1,93 @@
...
+            grid-area: contact;
+            width: 100%;
+            height: 100%;
+        }
+
+        .accountWrapper__recentTransaction{
```
Comment (ChrisWakley) [inline src/pages/AccountPage/AccountPage.scss:55]: &__recentTransaction

File: src/components/BankInfoSqure/BankInfoSquare.jsx [hunk 19]
```diff
@@ -0,0 +1,32 @@
+import "./BankInfoSquare.scss";
+
+
+
+const BankInfoSqaure = ({firstName,lastName,img,sortCode,accNum,iban}) => {
```
Comment (kirsty-nology) [inline src/components/BankInfoSqure/BankInfoSquare.jsx:5]: folder name is spelled incorrectly, component also spelt incorrectly

File: src/pages/AccountPage/AccountPage.jsx [hunk 20]
```diff
@@ -0,0 +1,43 @@
...
+const AccountPage = ({firstName = "Erik", lastName = "Drew"}) => { 
+
+    return (
+        <>
+            <div className="accountWrapper">
+                <div className="loggedIn">
+                    <LoggedIn/>
+                </div>
+                    <div>
+                        <p className="accountWrapper__contacts">Contacts / <span className="changeColor">{firstName}{lastName}</span> </p>
```
Comment (kirsty-nology) [inline src/pages/AccountPage/AccountPage.jsx:18]: `className="account__contacts"`

File: src/pages/AccountPage/AccountPage.jsx [hunk 21]
```diff
@@ -0,0 +1,43 @@
...
+                        <p className="accountWrapper__heading">Account Details</p>
```
Comment (kirsty-nology) [inline src/pages/AccountPage/AccountPage.jsx:19]: `className="account__heading"`

File: src/pages/AccountPage/AccountPage.jsx [hunk 22]
```diff
@@ -0,0 +1,43 @@
...
+                    </div>
+                <div className="accountWrapper__info">
+                    <div className="accountWrapper__singleContact">
```
Comment (kirsty-nology) [inline src/pages/AccountPage/AccountPage.jsx:22]: `className="account__singleContact"`

File: src/pages/AccountPage/AccountPage.jsx [hunk 23]
```diff
@@ -0,0 +1,43 @@
...
+                        <BankInfoSqaure />
+                    </div>
+                    <div className="accountWrapper__transferRow">
```
Comment (kirsty-nology) [inline src/pages/AccountPage/AccountPage.jsx:25]: `className="account__transferRow"`

File: src/pages/AccountPage/AccountPage.jsx [hunk 24]
```diff
@@ -0,0 +1,43 @@
...
+                        <p>Want to make a transfer?</p>
+                        <Link to={"/Transfer"}><Button buttonText={"Transfer"} buttonClass={"corneredButton"}/></Link>
+                    </div>
+                    <div className="accountWrapper__recentTransaction">
```
Comment (kirsty-nology) [inline src/pages/AccountPage/AccountPage.jsx:29]: `className="account__transactions"`

File: src/App.jsx [hunk 25]
```diff
@@ -1,16 +1,16 @@
...
-import {useState} from 'react';
+import { useState } from "react";
 import Home from "./pages/Home/Home";
 import "./styles/main.scss";
 import Login from "./pages/Login/Login";
 import SignUp from "./pages/SignUp/SignUp";
 import Wallet from "./pages/Wallet/Wallet";
 import Nav from "./pages/Nav/Nav";
 
-const App = () => {
```
Comment (kirsty-nology) [inline src/App.jsx:10]: remove

File: src/App.jsx [hunk 26]
```diff
@@ -27,8 +27,9 @@ const App = () => {
           } 
         /> 
       )}
+
```
Comment (kirsty-nology) [inline src/App.jsx:30]: remove

File: src/pages/AccountPage/AccountPage.scss [hunk 27]
```diff
@@ -0,0 +1,93 @@
...
+            grid-area: transactions;
+            width: 100%;
+            height: 100%;
+        }
+    }
+}
+.changeColor{
+
```
Comment (kirsty-nology) [inline src/pages/AccountPage/AccountPage.scss:63]: remove line

**Review by kirsty-nology — CHANGES_REQUESTED** [review]

**Review by ChrisWakley — CHANGES_REQUESTED** [review]
please action changes.

**Review by kirsty-nology — APPROVED** [review]

**Review by lanniiee — APPROVED** [review]

**Review by ChrisWakley — APPROVED** [review]

**Pull request merged**